lib/log.js

```diff
@@ -0,0 +1,49 @@
+const bunyan = require('bunyan');
+
+const { name, version } = require('../package');
+const config = require('./config');
+
+const options = {
+  name,
+  level: config.LOG_DISABLE ? (bunyan.FATAL + 1) : config.LOG_LEVEL, // turn off logging https://github.com/trentm/node-bunyan/pull/148#issuecomment-63018668
+  version,
+  src: true,
+  obfuscate: [
+    'body.password'
+  ],
+  period: '1d', // daily rotation
```

Comment:
Daily might be to often, especially for small like apps

Reply:
it will be moved to config anyway